setup.cfg

```diff
@@ -41,8 +41,14 @@ install_requires =
     qtpy
     n2v>=0.3.1
     bioimageio.core
-    pyqtgraph,
-    napari_time_slicer
+    pyqtgraph
+    napari_time_slicer >= 0.4.9
+    napari_tools_menu >= 0.1.17
```

Comment:
Since `napari_tools_menu` is a dependency of `napari_time_slicer`, is there a reason to add it here, except to pin the dependency?

Reply:
Oh, is it? Cool! Then, it's really not necessary. Good catch!